general/payment_acquirers/authorize.rst

```diff
@@ -1,81 +1,116 @@
-  by clicking 'Get Started'.
-* In the pricing page, press *Sign up now* if you want to use Authorize.net as
-  both payment gateway and merchant. If you want to use your own merchant, press
-  the related option.
+Authorize.Net account
+=====================
 
-  .. image:: media/authorize01.png
-    :align: center
+If not done yet, choose a plan and `Sign Up for an Authorize.Net account
+<https://www.authorize.net/sign-up.html>`__.
 
-* Go through the registration steps. 
-* The account is set as a test account by default. You can use this test
-  account to process a test transaction from Odoo.
-* Once ready, switch to **Production** mode. 
+Odoo needs your **API Credentials & Keys** to connect with your Authorize.Net account, which
+comprise:
 
+- API Login ID
+- Transaction Key
+- Signature Key
 
-Set up Odoo
-===========
-* Activate Authorize.Net in Odoo from :menuselection:`Website or Sales or Accounting 
-  --> Settings --> Payment Acquirers`.
-* Enter both your **Login ID** 
-  your **API Transaction Key** and your **API Signature Key**.
+To retrieve them, log into your Authorize.Net account, go to :menuselection:`Account --> Security
+Settings --> General Security Settings --> API Credentials & Keys`, and generate your **Transaction
+Key** and **Signature Key**.
 
-  .. image:: media/authorize02.png
-    :align: center
+.. image:: media/authorize-api-keys.png
+   :align: center
+   :alt: Generate your Transaction Key and Signature Key on your Authorize.Net account
 
-  To get those credentials from Authorize.Net, you can follow the direction
-  in the Authorize.net Help
-  `API Credentials and Keys <https://account.authorize.net/help/Account/Settings/Security_Settings/General_Settings/API_Login_ID_and_Transaction_Key.htm>`__
-  entry and access the menu :menuselection:`Account --> Settings --> API Credentials & Keys`
-  in the Authorize.net dashboard. Note that you need *both* a Transaction key and and Signature key.
+.. seealso::
 
-* Once both keys are set up, you can then generate a Client Key by clicking the
-  "Generate Client Key" button. Note that these 3 keys are necessary for the
-  setup to work correctly.
+   - `Authorize.Net: Getting Started Guide
+     <https://support.authorize.net/s/article/Authorize-Net-Getting-Started-Guide>`__
 
+Payment Acquirer Configuration
+==============================
 
-Go live
-=======
-Your configuration is now ready! 
-You can make Authorize.Net visible on your merchant interface
-and activate the **Enabled** state.
+To configure Authorize.Net as Payment Acquirer in Odoo, go to :menuselection:`Accounting -->
+Configuration --> Payment Acquirers`, open **Authorize.Net**, and change the **State** to *Enabled*.
+Don't forget to click on *Save* once you've set everything up.
 
-.. image:: media/paypal_live.png
-    :align: center
+.. note::
+   Please refer to the :doc:`Payment Acquirers documentation <payment_acquirers>` to read how to
+   configure this payment acquirer.
```

Comment:
This link is already available above. Not sure it is useful to have it several times in the same doc.

Reply:
It's a different section and it specifically directs readers to configure according to that doc, so I'd keep it here, at least. But I think it's okay to have it twice in the same doc. I don't expect everyone to read everything from top to bottom.

Cheers :)